exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -159,6 +163,14 @@ public void start() throws Exception {
       servletContextHandler.setSessionHandler(createSessionHandler(servletContextHandler.getSecurityHandler()));
     }
 
+    if (config.getBoolean(ExecConstants.HTTP_ENABLE_CORS)) {
+      FilterHolder cors = servletContextHandler.addFilter(CrossOriginFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
```

Comment:
Not so much experience regarding CORS, but do we want to apply it to any resource, or just the REST API, or maybe a subset of those?